operator/Makefile

```diff
@@ -7,6 +7,14 @@
 # dropped (e.g. 3.0.61.1 -> 3.61.1) and development version ".x" is changed to ".0" (e.g. 3.0.61.x-123 -> 3.0.61.0-123).
 VERSION ?= $(shell $(MAKE) --quiet --no-print-directory -C .. tag | sed -E 's@^(([[:digit:]]+\.)+)x(-)?@\10\3@g' | sed -E 's@^3.0.([[:digit:]]+\.[[:digit:]]+)(-)?@3.\1\2@g')
 
+# The operator version used by ClusterServiceVersion object and related tooling (bundle, index). It is prefixed with "v".
+CSV_VERSION ?= v$(VERSION)
+# Operator index image tag to install from, used by deploy*-via-olm targets.
+INDEX_IMG_TAG ?= $(CSV_VERSION)
+# Operator version to install and channel to install from, used by deploy-(dirty-tag-)via-olm targets.
+INSTALL_VERSION ?= $(CSV_VERSION)
+INSTALL_CHANNEL ?= latest
```

Comment:
If we are to keep `INSTALL_VERSION` and `INSTALL_CHANNEL`, I suggest making their designation more clear.
At least something along the lines of
```suggestion
# Overridable operator version and channel to install for (deploy|upgrade)*-via-olm targets.
INSTALL_VERSION ?= $(CSV_VERSION)
INSTALL_CHANNEL ?= latest
```

If you can come up with more specific names, it will be better.